amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -4307,26 +4321,100 @@ definitions:
+          * year
       period:
         type: string
+        description: |
+          Valid values for the period.
+          * day = Day
+          * fmend = End Of Financial Month
+          * fqend = End Of Financial Quarter
+          * fyend = End Of Financial Year
+          * immediately = Immediately
+          * monthend = End Of Month
+          * quarterend = End Of Quarter
+          * yearend = End Of Year
+          * month = Month
+          * none = None
+          * quarter = Quarter
+          * week = Week
+          * duration = XML Duration
+          * year = Year
       periodProperty:
         type: string
+        default: cm:created
+        description: |
+          Valid values for the periodProperty property
+          * cm:created = Created Date (defult value) 
+          * rma:cutOffDate = Cut Off Date
+          * rma:dispositionAsOf = Retention Action
       combineDispositionStepConditions:
         type: boolean
+        description: |
+          This property is only valid for **accession** step.
+          This is used to specify whether to combine the period condition and events for the step execution or only consider one of them.
+          For example:
+          **periodCondition**: After a period of 2 months
+          **eventsCondition**: Case Closed event
+          This flag can be used to consider only (**periodCondition** or **eventsCondition**) or both of them at once.
       events:
         type: array
         items:
           type: string
+        description: |
+          Valid values for the events property
+          * case_closed = Case Closed
+          * abolished = Abolished
+          * re_designated = Redesignated
+          * no_longer_needed = No Longer Needed
+          * superseded = Superseded
+          * versioned = Versioned
+          * study_complete = Study Complete
+          * training_complete = Training Complete
+          * related_record_trasfered_inactive_storage = Related Record Transferred to Inactive Storage
+          * obsolete = Obsolete
+          * all_allowances_granted_are_terminated = All Allowances Granted are Terminated
+          * WGI_action_complete = WGI Action Complete
+          * separation = Separation
+          * case_complete = Case Complete
+          * declassification_review = Declassification Review
       eligibleOnFirstCompleteEvent:
         type: boolean
+        description: |
+          * false = When all events have happened
+          * true = Whichever event is earlier
       description:
         type: string
-      destroyWithGhosting:
+        description: |
+          This property is used to provide the step description.
+      destroyNode:
+        type: boolean
+        description: |
+          This property is used to destroy the node along with metadata.
+      destroyContent:
         type: boolean
+        description: |
+          This property is used to destroy the content but node metadata will preserved.
```

Comment:
I think these references to destroyNode and destroyContent can go too.

Reply:
As we removed above properties so updated the documentation along with it.